invenio_github/views/badge.py

```diff
@@ -73,12 +73,15 @@ def index(repo_github_id):
 @blueprint.route("/<int:user_id>/<path:repo_name>.svg")
 def index_old(user_id, repo_name):
     """Generate a badge for a specific GitHub repository."""
-    github_api = GitHubAPI(user_id)
-    repo = github_api.get_repository(repo_name=repo_name)
-    release = github_api.repo_last_published_release(repo)
-    if not release:
+    repo = Repository.query.filter(Repository.name == repo_name).one_or_none()
```

Comment:
This view in [legacy](https://github.com/inveniosoftware/invenio-github/blob/v1.0.0a28/invenio_github/views/badge.py#L75) discarded the `user_id` parameter and simply rendered the badge. 

Since we're getting many `User can't access repository`, I made the change for the endpoint to be exactly like it was in legacy.